src/gc/cdgc/gc.d

```diff
@@ -43,7 +43,6 @@ version = STACKGROWSDOWN;       // growing the stack means subtracting from the
 /***************************************************/
 
 import gc.cdgc.bits: GCBits;
-import gc.cdgc.stats: GCStats, Stats;
```

Comment:
Yeah I don't plan to really merge it, this PR is only used to generate private docker image for investigations.